sdk/keyvault/keyvault-keys/src/cryptography/crypto.browser.ts

```diff
@@ -27,7 +27,7 @@ export function createVerify(_algorithm: string, _data: Uint8Array): never {
  * @internal
  * Use the platform-local randomBytes functionality
  */
-export function randomBytes(_length: number): Uint8Array {
+export function randomBytes(_length: number): never {
```

Comment:
Is this change related to this PR? Is it because it’s simply not really supported in the browser?

Reply:
Yeah, just cleaning up some internal functions. `never` seems appropriate here